app/src/main/java/com/google/samples/apps/sunflower/compose/garden/GardenScreen.kt

```diff
@@ -64,13 +64,15 @@ import java.util.*
 
 @Composable
 fun GardenScreen(
+    modifier: Modifier = Modifier,
     viewModel: GardenPlantingListViewModel = viewModel(),
     onAddPlantClick: () -> Unit,
     onPlantClick: (PlantAndGardenPlantings) -> Unit
 ) {
     val gardenPlants by viewModel.plantAndGardenPlantings.collectAsState(initial = emptyList())
     GardenScreen(
```

Comment:
I created 1 with a view model dependency and another w/o one. The latter is used to create a preview for the `GardenScreen` so that I don't have to mock/fake a view model (it's a pattern I generally followed in the sample app).